packages/services/f-feature-management/package.json

```
@@ -41,7 +41,7 @@
     "eslint-plugin-json-schema-validator": "1.0.48",
     "eslint-plugin-jsonc": "1.3.1",
     "jest": "25.4.0",
-    "jest-fetch-mock": "3.0.3",
+    "jest-mock-axios": "^4.4.1",
```

It's standard within this repo to pin to a specific version if possible:
```suggestion
    "jest-mock-axios": "4.4.1",
```